packages/atomic/src/components/common/smart-snippets/atomic-smart-snippet-collapse-wrapper/atomic-smart-snippet-collapse-wrapper.tsx

```diff
@@ -13,17 +16,33 @@ export class AtomicSmartSnippetCollapseWrapper {
   public error!: Error;
   @Element() public host!: HTMLElement;
 
-  @Prop({reflect: true}) public maximumHeight?: number;
+  @Prop({reflect: true}) public snippetMaximumHeight?: number;
 
-  @Prop({reflect: true}) public collapsedHeight?: number;
+  @Prop({reflect: true}) public snippetCollapsedHeight?: number;
```

Comment:
Right, because of yassine's comment. I thought it would be best to keep the same name across the two components but you are right. This doesnt make much sense. I will change it back. thanks.